docs/config-system.md

```diff
@@ -53,24 +51,24 @@ If the value for a field stored in on disk somehow ends up being invalid the fie
 
 ## Accessing Fields
 
-On the Config instance there's two ways to access fields: through the set and get methods using dot notation.
+On the Config instance there's two ways to access fields: through the set and get methods.
 E.g.:
 
     let value = configInstance.get("group.field");
 
-Or by getting the group first and then calling set or get on that.
+Or by through the raw fields object (which is best avoided as it doesn't check types or whether the accessed field exists).
 E.g.:
 
-    let value = configInstance.group("group").get("field")
+    let value = configInstance.fields["group.field"]
```

Comment:
I don't think it hurts to leave it accessible. It doesn't need to be private or hidden behind a function because the whole idea behind the config refactor was to make it simpler.